src/interfaces/IUniformRandom.sol

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: BUSL 1.1
+// Valorem Labs Inc. (c) 2022
+pragma solidity 0.8.16;
+
+/**
+ * @notice An interface for a random variable uniformly distributed over
+ * the half open range supplied.
+ */
+interface IUniformRandom {
```

Comment:
I'm not sure we need an interface for this per se, did you see @neodaoist's PR with the randomness source?